site/class-1-bar/kyunwang/index.js

```diff
@@ -198,6 +198,10 @@ function sortData(data, typeSort) {
 d3.selectAll('#colorChange')
 	.on('click', function () {
 
+<<<<<<< HEAD
+		console.log(colorIndex);
+=======
+>>>>>>> 6a8509732bd20453bdf9aea7490e639fedc1a226
```

Comment:
Your clean assignment looks great, but you’re proposing breaking changes to your bar chart.
Seems you have this on your master branch? Could you remove your fork, fork again, and upload clean again?